pkgdb/src/main.cc

```diff
@@ -76,7 +76,10 @@ setVerbosityFromEnv()
   auto * valueChars = std::getenv( "_FLOX_PKGDB_VERBOSITY" );
   if ( valueChars == nullptr ) { return; }
   std::string value( valueChars );
-  if ( value == std::string( "0" ) ) { nix::verbosity = nix::lvlError; }
+  if ( value == std::string( "-1" ) || value == std::string( "0" ) )
+    {
+      nix::verbosity = nix::lvlError;
+    }
```

Comment:
Suggestion handle Level 0 as warn/default

Reply:
That would be a behavior change. Are you sure there aren't warnings that would start displaying for plain `flox` commands that we don't want?

Reply:
If this is a blocker can you re-raise on https://github.com/flox/flox/pull/2410?